services/endpoint/endpoint.tester.js

```diff
@@ -30,11 +30,8 @@ t.create('color and labelColor')
         labelColor: '#e6e6fa',
       })
   )
-  .expectJSON({
-    name: 'hey',
-    value: 'yo',
-    color: '#f0dcc3',
-    labelColor: '#e6e6fa',
+  .after(body => {
+    expect(body).to.include('fill="#e6e6fa"')
```

Comment:
I haven’t re-read all these tests, though judging from the test title I think this needs to test `color` as well. If you add `labelColor` to the JSON body, you could revert this to be a JSON test.

Reply:
Extended `expectBadge` to handle `labelColor` and `logoWidth`.